frontend/apps/web/app/(mgmt)/[account]/runs/[id]/page.tsx

```diff
@@ -409,7 +409,8 @@ function ViewSelectDialog(props: ViewSelectDialogProps): ReactElement {
 
   const formattedQuery = useMemo(() => {
     // todo: maybe update this to explicitly pass in the driver type so it formats it according to the correct connection
-    return formatSql(query.select);
+    // return formatSql(query.select);
+    return query.select;
```

Comment:
Why was this changed? Maybe unintentionally?

Reply:
i couldn't get the sql formatter to import and work correctly so it was a temp change until i could figure it out.. then you mentioned that you have to npm install it into the container (or somethign like that) and i just hadn't changed it back. Changed it now and seems to be working.